index.html

```diff
@@ -110,13 +109,13 @@ <h4>Introduction</h4>
 
           Some expressions have side effects:
```

Comment:
This was only relevant to Clojure. In Haskell things can be used before they are defined.